modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/PlacementDriverManager.java

```diff
@@ -31,21 +46,94 @@ public class PlacementDriverManager implements IgniteComponent {
+     */
+    private final CompletableFuture<TopologyAwareRaftGroupService> raftClientFuture;
+
+    private final ScheduledExecutorService raftClientExecutor;
+
+    private final LogicalTopologyService logicalTopologyService;
+
+    private final RaftConfiguration raftConfiguration;
+
+    private volatile boolean isActiveActor;
+
+    private volatile long lastTermSeen = -1;
+
     /**
      * The constructor.
      *
-     * @param metaStorageMgr Meta Storage manager.
+     * @param replicationGroupId Id of placement driver group.
+     * @param clusterService Cluster service.
+     * @param raftConfiguration Raft configuration.
+     * @param placementDriverNodesNamesProvider Provider of the set of placement driver nodes' names.
+     * @param logicalTopologyService Logical topology service.
+     * @param raftClientExecutor Raft client executor.
      */
-    public PlacementDriverManager(MetaStorageManager metaStorageMgr) {
+    public PlacementDriverManager(
+            ReplicationGroupId replicationGroupId,
+            ClusterService clusterService,
+            RaftConfiguration raftConfiguration,
+            Supplier<CompletableFuture<Set<String>>> placementDriverNodesNamesProvider,
+            LogicalTopologyService logicalTopologyService,
+            ScheduledExecutorService raftClientExecutor
+    ) {
+        this.replicationGroupId = replicationGroupId;
+        this.clusterService = clusterService;
+        this.raftConfiguration = raftConfiguration;
+        this.placementDriverNodesNamesProvider = placementDriverNodesNamesProvider;
+        this.logicalTopologyService = logicalTopologyService;
+        this.raftClientExecutor = raftClientExecutor;
+
+        raftClientFuture = new CompletableFuture<>();
     }
 
     /** {@inheritDoc} */
     @Override
     public void start() {
+        placementDriverNodesNamesProvider.get()
+                .thenCompose(placementDriverNodes -> {
+                    String thisNodeName = clusterService.topologyService().localMember().name();
+
+                    if (placementDriverNodes.contains(thisNodeName)) {
+                        return TopologyAwareRaftGroupService.start(
+                                replicationGroupId,
+                                clusterService,
+                                raftMessagesFactory,
+                                raftConfiguration,
+                                PeersAndLearners.fromConsistentIds(placementDriverNodes),
+                                true,
+                                raftClientExecutor,
+                                logicalTopologyService,
+                                true
+                            ).thenCompose(client -> {
+                                TopologyAwareRaftGroupService topologyAwareClient = (TopologyAwareRaftGroupService) client;
 
+                                return topologyAwareClient.subscribeLeader(this::onLeaderChange).thenApply(v -> topologyAwareClient);
```

Comment:
If we subscribe to the leader elected event, where will we unsubscribe?
I think it should be before stop (IgniteComponent#beforeNodeStop).